elements/pfe-cta/demo/demo.css

```diff
@@ -18,33 +18,26 @@ pfe-band[pfe-color="lightest"] pfe-card[pfe-color="lightest"] {
 }
 
 .resize {
-  display: flex;
-  flex-flow: row nowrap;
   resize: horizontal;
   overflow: auto;
   border: 1px solid #ddd;
-  width: 500px;
   padding: 10px;
 }
 
-.resize > pfe-cta:first-child {
-  margin-right: 20px;
-}
-
 .card-layout {
   display: flex;
   flex-flow: row wrap;
   display: grid;
   grid-gap: 10px;
-  grid-template-columns: repeat(auto-fit, minmax(100px, max-content));
+  grid-template-columns: repeat(auto-fit, minmax(280px, max-content));
   margin-top: -10px;
   margin-left: -20px;
 }
 
 .card-layout > pfe-cta,
 .card-layout .cta-align > pfe-cta {
   margin-top: 10px;
-  margin-left: 20px;
```

Comment:
This is breaking the card-layout in the custom bands below - it needs to compensate for the `margin-left: -20px` above.

Reply:
Why set the CTAs to `display: block`?  Don't we want to test that the CTA's display mode is correct?  This would override the display mode for the host element.

Reply:
The second point is fair. I was trying to make the demo prettier but I think its more important to show the native behavior.